ui/imports/utils/Constants.qml

```diff
@@ -418,6 +418,9 @@ QtObject {
                 readonly property string watchOnlyAccounts: "watchOnlyAccounts"
             }
         }
+
+        readonly property int defaultPinAttempts: 3
```

Comment:
is it `defaultPinAttempts` or just `pinAttempts` as it's the only valid value to be used across the app?

Reply:
I used the "default" prefix to differentiate it from the actual remaining PIN attempts number